shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/metadata/model/PipelineTableMetaDataTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.shardingsphere.data.pipeline.core.metadata.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Types;
+import java.util.List;
+import java.util.Map;
+
+public final class PipelineTableMetaDataTest {
+    private PipelineTableMetaData pipelineTableMetaData;
+    private Map<String, PipelineColumnMetaData> columnMetaDataMap;
+
+    @Before
+    public void setUp() {
+        PipelineColumnMetaData pipelineColumnMetaData = new PipelineColumnMetaData(0, "test", Types.INTEGER, "INTEGER", true);
+        columnMetaDataMap.put("test_column", pipelineColumnMetaData);
+        pipelineTableMetaData = new PipelineTableMetaData(null, columnMetaDataMap);
+    }
+
+    @Test
+    public void assertGetColumnMetaDataGivenColumnIndex() {
+        List<String> pipelineTableColumns = pipelineTableMetaData.getColumnNames();
+        PipelineColumnMetaData actual = columnMetaDataMap.get(pipelineTableColumns.get(0));
+        assertThat(actual.getOrdinalPosition(), is(0));
```

Comment:
1. It's better to invoke `pipelineTableMetaData.getColumnMetaData` instead of `columnMetaDataMap.get(pipelineTableColumns.get(0))`, since the logic is encapsulated in `pipelineTableMetaData` and we'll test it.

2. `columnMetaDataMap` could be hidden in `setUp`.

Reply:
1. Yes agreed, I have made the change.
2. Shifted `columnMetaDataMap` declaration to `setUp`

Reply:
I have done away with `columnMetaDataMap`. This is how `setUp` method looks now

```java
    @Before
    public void setUp() {
        pipelineTableMetaData = new PipelineTableMetaData(null, Collections.singletonMap("test_column", new PipelineColumnMetaData(0, "test", Types.INTEGER, "INTEGER", true)));
    }
```

Reply:
Nice. You could commit the change and push again, I'll enable the CI workflow then.